ui.frontend/src/main/webpack/components/_footer.scss

```diff
@@ -0,0 +1,47 @@
+.cmp-layoutcontainer--footer {
+  display: flex;
+  justify-content: center;
+  background-color: $color-white;
+
+  .cmp-layoutcontainer--footer__wrapper {
```

Comment:
Just as a recommendation, use scss capabilities here. When working with BEM practices, we can take advantage of nesting code using '&'